armory/scenarios/gtsrb_scenario.py

```diff
@@ -0,0 +1,170 @@
+"""
+Classifier evaluation within ARMORY
+"""
+
+import json
+import os
+import sys
+import logging
+import coloredlogs
+from importlib import import_module
+
+import numpy as np
+
+from armory.utils.config_loading import load_dataset, load_model
+from armory import paths
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+coloredlogs.install(logging.INFO)
```

Comment:
During PR for Scenario subclass we should remove the logger level being set here